TestPullRequest/src/testPackage/Calc.java

```diff
@@ -2,9 +2,9 @@
 
 public class Calc extends SuperCalc implements CalcInterface {
 	
-	public final static int MULTIPLIER = 10;
+	public static final int MULTIPLIER = 10;
 
-	public Integer addition(int a, int b, int c) {
+	public Integer addieren(int a, int b, int c) {
```

Comment:
@LHommeDeBot Could you rename this method to 23b407d3h2ea513a9856c87f672410272560d62a